File: stepik-union/src/main/java/org/stepik/plugin/refactoring/impl/idea/rename/RenamePsiElementProcessor.java
package org.stepik.plugin.refactoring.impl.idea.rename;

import org.stepik.plugin.refactoring.impl.idea.AcceptableClasses;
import org.stepik.plugin.refactoring.rename.AbstractRenamePsiElementProcessor;

/**
 * @author meanmail
 */
public class RenamePsiElementProcessor extends AbstractRenamePsiElementProcessor {
    public RenamePsiElementProcessor() {
        addAcceptableClasses(AcceptableClasses.get());
    }
}

File: stepik-union/src/main/java/org/stepik/plugin/refactoring/impl/idea/rename/RenamePsiElementProcessor.kt
package org.stepik.plugin.refactoring.impl.idea.rename

import org.stepik.plugin.refactoring.impl.idea.AcceptableClasses
import org.stepik.plugin.refactoring.rename.AbstractRenamePsiElementProcessor


class RenamePsiElementProcessor : AbstractRenamePsiElementProcessor() {
    init {
        addAcceptableClasses(AcceptableClasses.get())
    }
}
